src/lib/Behat/Component/TranslationDialog.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Behat\Component;
+
+use Ibexa\Behat\Browser\Element\Criterion\ElementTextCriterion;
+use Ibexa\Behat\Browser\Locator\VisibleCSSLocator;
+
+class TranslationDialog extends Dialog
+{
+    protected function chooseFromDropdown(string $language): void
+    {
+        $this->getHTMLPage()
+            ->findAll($this->getLocator('dropdownItem'))
+            ->getByCriterion(new ElementTextCriterion($language))
+            ->click();
+    }
+
+    public function selectNewTranslation(string $value): void
+    {
+        $this->getHTMLPage()->find($this->getLocator('expandNewTranslationDropdown'))->click();
+        $this->chooseFromDropdown($value);
+    }
```

Comment:
```suggestion
    public function selectNewTranslation(string $languageName): void
    {
        $this->getHTMLPage()->find($this->getLocator('expandNewTranslationDropdown'))->click();
        $this->chooseFromDropdown($languageName);
    }
```

`value` feels too generic here